saray/message-page/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="styles.css" />
+    <link href="https://fonts.googleapis.com/css?family=Lato:200,300,400,100,900" rel="styles">
+    <title>Message Page</title>
+</head>
+<body>
+   <div class="view-port">
+        <div class="container">
+            <div class="message-image">
```

Comment:
I think that you don't need put the```<div> </div>``` element. You can to use the element ```<img>``` directly